data/xml/Q13.xml

```diff
@@ -35,6 +35,7 @@
       <pages>13–24</pages>
       <url hash="84ba73ea">Q13-1002</url>
       <bibkey>pitler-etal-2013-finding</bibkey>
+      <video tag="video" href="Q13-1002.mp4"/>
```

Comment:
I'm not sure why we keep this `tag="video"` around. I've done a grep on all our current XML files and the value for `tag` is never something else than `"video"`. Since we also don't do anything with it in the code, I think we should just get rid of it now. @mjpost?

For reference:
```
$ grep "<video" *.xml | sed 's/.*tag="\([[:alpha:]]*\)".*/\1/' | sort | uniq -c                                                                                                                                 
   4010 video
```

Reply:
Yes, @xinru1414, let’s get rid of `tag=“video”`. We’ve had this discussion elsewhere before.

I also agree it might be good to use a dictionary for video values. For example, we might want to have a field that distinguishes remote from local links. We don’t need it now so we could just refactor later, but it might be easier now to preserve the dictionary structure while it’s fresh in your memory.

Reply:
When getting rid of the tag, remember to update the schema and remove tags from the video element here: https://github.com/acl-org/acl-anthology/blob/3ef7ced577d9c95feaf2cecc94cd33d13398eea1/data/xml/schema.rnc#L72

Reply:
@akoehn yep SG thanks for the reminder